www/guides/react.md

```diff
@@ -0,0 +1,373 @@
+---
+layout: layouts/guide.njk
+permalink: '/guides/react/'
+title: Snowpack + React Guide
+description: 'Documentation and guides for building projects with Snowpack and React.'
+date: 2020-10-01
+---
+
+Image: ideally a nice header image showing the before or after, or some illustration with elements of the Snowpack logo, React, and symbols related to speed, joy.
+
+Snowpack is a great fit for [React](https://reactjs.org/) projects of any size. It's easy to get started with and can scale to projects containing thousands of components and pages without any impact on development speed. Unlike traditional React application tooling, Snowpack saves you from getting bogged down with complex bundler setups and configuration files.
+
+In this guide, you'll go from a [minimal Snowpack project](https://github.com/pikapkg/snowpack/tree/master/create-snowpack-app/app-template-minimal) to a Snowpack project with React and several useful developer optimizations. In the process you'll learn:
+
+- How to set up a new Snowpack project
+- How to get started with React in a Snowpack project
+- Importing and using CSS and other assets inside your project
```

Comment:
run-on sentence. How about "Importing CSS, Sass, Images and other assets"

(this would work if we had a mention for Sass somewhere in the CSS section. We get so many people asking about Sass, I think it makes sense to have a mention).